influxql/iterator.gen.go

```diff
@@ -2297,6 +2304,7 @@ func (itr *floatBooleanExprIterator) Next() (*BooleanPoint, error) {
 			if itr.points == nil {
 				continue
 			}
+
```

Comment:
yeah, I have no idea what that would be....  I don't even know what this method even does.  Comments FTW :-).

I would need your help on this to figure it out.